docs/user_guides/fs/feature_view/feature_logging.md

```diff
@@ -0,0 +1,208 @@
+
+You can log either transformed or/and untransformed features. To get untransformed features, you can specify `transform=False` in `feature_view.get_batch_data` or `feature_view.get_feature_vector(s)`. Inference helper columns are returned along with the untransformed features. To get the transformed features, you can call `feature_view.transform_batch_data` or `feature_view.transform_feature_vector(s)`. Inference helper columns are not returned as transformed features. [link to transformed features]()
+
+You can also log predictions, and optionally the training dataset version and the model used for prediction. Prediction can be optionally provided as a column in the feature DataFrame or separately in the `prediction` argument. This is useful for logging real-time features and predictions which are often in type `list`, avoiding the need to ensure feature order of the labels. Training dataset version will also be logged if it is cached after you provide the training dataset version when calling  `feature_view.init_serving(...)` or `feature_view.init_batch_scoring(...)`.
+
+The time of calling `feature_view.log` is automatically logged, enabling filtering by logging time when retrieving logs.
+
+#### Example 1: Log Features Only
+
+You have a DataFrame of features you want to log.
+
+```python
+import pandas as pd
+
+features = pd.DataFrame({
+    "feature1": [1.1, 2.2, 3.3],
+    "feature2": [4.4, 5.5, 6.6]
+})
+
+# Log features
+feature_view.log(features)
+```
+
+#### Example 2: Log Features, Predictions, and Model
+
+You can also log predictions, and optionally the training dataset and the model used for prediction.
+
+```python
+predictions = pd.DataFrame({
+    "prediction": [0, 1, 0]
+})
+
+# Log features and predictions
+feature_view.log(features, 
+                 prediction=predictions, 
+                 training_dataset_version=1, 
+                 hsml_model=Model(1, "model", version=1)
+)
+```
+
+#### Example 3: Log Both Transformed and Untransformed Features
+
+**Batch Features**
+```python
+untransformed_df = fv.get_batch_data(transform=False)
+# then apply the transformations after:
+transformed_df = fv.transform_batch_data(untransformed_df)
+# Log untransformed features
+feature_view.log(untransformed_df)
+# Log transformed features
+feature_view.log(transformed_df, transformed=True)
+```
+
+**Real-time Features**
+```python
+untransformed_vector = fv.get_feature_vector({"id": 1}, transform=False)
+# then apply the transformations after:
+transformed_vector = fv.transform_feature_vector(untransformed_vector)
+# Log untransformed features
+feature_view.log(untransformed_vector)
+# Log transformed features
+feature_view.log(transformed_vector, transformed=True)
+```
+
+## Retrieving the Log Timeline
+
+To audit and review the data logs, you might want to retrieve the timeline of log entries. This helps understand when data was logged and monitor the logging process.
+
+### Retrieve Log Timeline
```

Comment:
This is not a "log timeline" as i understand it.
For me a "log timelime" is about the log itself - the log was created, it was rotated, it was deleted.
It is not about the log entries, which is what we are interested in here.

Reply:
It is actually the hudi commit timeline.